### PR TITLE
Songbird recipe

### PR DESCRIPTION
**Pull request by mortonjt**
Thank you for this awesome resource!  Adding in a recipe for songbird (found here: https://github.com/biocore/songbird) to make this package more widely available.

### CODE REVIEW

File: recipes/songbird/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{% set version = "0.8.2" %}
+
```
Comment (mwcraig) [inline recipes/songbird/meta.yaml]:
```suggestion
{% set name = "songbird" %}

```

File: recipes/songbird/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+package:
+  name: songbird
```
Comment (mwcraig) [inline recipes/songbird/meta.yaml]:
```suggestion
  name: {{ name }}
```

File: recipes/songbird/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  version: {{ version }}
+
+source:
+  fn: songbird-{{ version }}.tar.gz
```
Comment (mwcraig) [inline recipes/songbird/meta.yaml]: You can drop the filename; it is no longer needed.

File: recipes/songbird/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  url: https://pypi.io/packages/source/s/songbird/songbird-{{ version }}.tar.gz
```
Comment (mwcraig) [inline recipes/songbird/meta.yaml]:
```suggestion
  url: https://pypi.io/packages/source/s/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipes/songbird/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python setup.py install --record=record.txt
+
+requirements:
+  build:
+    - python >=3.5
```
Comment (mwcraig) [inline recipes/songbird/meta.yaml]: I'm guessing you can drop most of the build requirements except python and pip.

File: recipes/songbird/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+  sha256: 1e152e5d6114135d0299a362ffd082ca525a6b4f8fc0cc1891ea131892e10a37
+
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python setup.py install --record=record.txt
+
+requirements:
+  build:
```
Comment (mwcraig) [inline recipes/songbird/meta.yaml]:
```suggestion
  host:
```

The name of this section has been changed.

File: recipes/songbird/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+source:
+  fn: songbird-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/songbird/songbird-{{ version }}.tar.gz
+  sha256: 1e152e5d6114135d0299a362ffd082ca525a6b4f8fc0cc1891ea131892e10a37
+
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python setup.py install --record=record.txt
```
Comment (mwcraig) [inline recipes/songbird/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

This is the preferred way of doing the installs these days

File: recipes/songbird/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+    - tqdm
+
+test:
+  imports:
+    - songbird
+
+
+about:
+  home: https://github.com/biocore/songbird
+  license: BSD 3-Clause
```
Comment (mwcraig) [inline recipes/songbird/meta.yaml]: Please also add license family and file, like at https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L62

File: recipes/songbird/meta.yaml (hunk 3 above)
Comment (mortonjt) [reply recipes/songbird/meta.yaml]: 👍

**Comment by mortonjt** [comment]
Hi @mwcraig thank you for reviewing this recipe!  It looks like most of the tests are passing now -- but there are tests that have been queued for the last few days.  Is this expected?

**Comment by mortonjt** [comment]
@conda-forge/help-python

**Comment by mwcraig** [comment]
@mortonjt -- I think the pypi.org download issue was temporary; I just checked and the pypi URL is working now.

This will be ready to go once you:

- [ ] Switch back to using the pypi URL 
- [ ] Add the license family
- [ ] Add the license file. You will need to include a copy in the recipe because it looks like it isn't included in the source distribution currently. More details are here: http://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-licence-manually

Appreciate your continued efforts on this and looking forward to merging soon!

**Comment by mortonjt** [comment]
cool - glad all of the changes are passing.  pypi URL is reverted and license file name is specified.

File: recipes/songbird/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+  license_family: BSD
```
Comment (mortonjt) [inline recipes/songbird/meta.yaml]: Are we still missing the license_family @mwcraig ?

Comment (mwcraig) [reply recipes/songbird/meta.yaml]: This is good

**Comment by mwcraig** [comment]
Thanks, @mortonjt, merging! I'd recommend including the license in the songbird source distribution in your next release and removing the license from the feedstock.

**Comment by mortonjt** [comment]
Thank you!

**Pull request merged**